packages/components/src/popover/index.tsx

```diff
@@ -139,6 +140,19 @@ const AnimatedWrapper = forwardRef(
 
 const slotNameContext = createContext< string | undefined >( undefined );
 
+const getPopoverFallbackContainer = () => {
+	let container = document.body.querySelector(
```

Comment:
@youknowriad: At the risk of asking a stupid question, does this reliance on a global document carry any consequences for usages of Popover within/across the iframe boundary of block editors?

Reply:
no, I don't think there's impact on the iframe slots here. This one is the fallback if there's no Slot defined while if we had to render something within the iframe previously we were forced to have a Slot.

Reply:
Maybe to be extra thorough, we could use the anchor or the floating's `ownerDocument`?